spark-etl/src/main/scala/geotrellis/spark/etl/config/DefaultEtlConf.scala

```diff
@@ -0,0 +1,87 @@
+package geotrellis.spark.etl.config
+
+import geotrellis.spark.etl.config.json._
+
+import org.apache.spark.SparkContext
+import com.github.fge.jackson.JsonLoader
+import spray.json._
+
+trait DefaultEtlConf extends ConfigParse {
```

Comment:
@lossyrob I think that the word base is better, though for some reason I chose Default (guess it's spray json influence )